evap/development/management/commands/translate.py

```diff
@@ -9,6 +9,10 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         self.stdout.write('Executing "manage.py makemessages --locale=de --ignore=node_modules/*"')
         call_command("makemessages", "--locale=de", "--ignore=node_modules/*")
+        self.stdout.write(
```

Comment:
Can you add a `logged_call_command` to `commands/tools.py` and use that here, something along the lines of
```python3
def logged_call_command(stdout, *args):
    stdout.write("Executing " + " ".join(args))
    call_command(*args)
```
so we don't have the arguments duplicated (and thus possibly diverging in the future)?